csharp/src/Ice/ObjectAdapter.cs

```diff
@@ -67,7 +67,7 @@ internal ThreadPool ThreadPool
             "AdapterId",
             "Endpoints",
             "Locator",
-            "Locator.EncodingVersion",
+            "Locator.Encoding",
```

Comment:
Should we keep the deprecated properties here?

Reply:
I don't think we need to. The use of EncodingVersion (for Locator and Ice.Default) should be extremely rare - for applications that dealt with encoding 1.0, so Ice <= 3.4.